src/replication.c

```diff
@@ -1666,6 +1740,9 @@ send_replica_query(replica_t *replica, spec_t *spec, zvol_op_code_t opcode)
 	zvol_op_code_t mgmt_opcode = opcode;
 	mgmt_cmd_t *mgmt_cmd;
 
+	if ((replica->state == ZVOL_STATUS_HEALTHY) && (opcode == ZVOL_OPCODE_REPLICA_STATUS))
```

Comment:
add a comment here
'This API sends the query to replica on its management connection. For healthy replicas, there is no need to send status queries'

Reply:
Comment is added.